module_fcst_grid_comp.F90

```diff
@@ -61,7 +61,9 @@ module module_fcst_grid_comp
                                     fv_srf_wnd_restart_bundle_setup, &
                                     fv_tracer_restart_bundle_setup
 
-  use fms2_io_mod,        only: FmsNetcdfFile_t, open_file, close_file, variable_exists, read_data
+  use fms2_io_mod,        only: FmsNetcdfFile_t, open_file, close_file, variable_exists, & 
+                                read_data
+  use fms_io_mod,         only: field_exist, read_data_old=>read_data
```

Comment:
Why is `read_data` subroutine from an old fms_io_mod needed? Can't `read_data` from  fms2_io_mod be used to read gridspec file used by these idealized tests?

Reply:
Good catch. This must have been a quick patch in testing. I've replaced its used with the standard read_data from fms2_io.